solidity/dashboard/src/pages/liquidity/LiquidityPage.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import PageWrapper from "../../components/PageWrapper"
+import CardContainer from "../../components/CardContainer"
+import LiquidityRewardCard from "../../components/LiquidityRewardCard"
+import { LIQUIDITY_REWARD_PAIR } from "../../constants/constants"
+import * as Icons from "../../components/Icons"
+
+const LiquidityPage = ({ title }) => {
+  return (
+    <PageWrapper title={title}>
+      <CardContainer className={"flex wrap"}>
```

Comment:
I think `CardContainer` should be responsible for wrapping elements and we shouldn't worry about it every time we use the` CardContainer` component.

Reply:
Fixed in https://github.com/keep-network/keep-core/pull/2216/commits/b0b960c8b0fbc53620cacdef5730474e8980c2a6